app/assets/stylesheets/mobile/compose.scss

```
@@ -33,6 +33,14 @@
     }
   }
 
+  // Firefox for Android hack
+  @supports (-moz-appearance: none) {
+    .keyboard-visible body:not(.ios-safari-composer-hacks) #reply-control.open {
```

I don't think `.ios-safari-composer-hacks` is a possibility here, so maybe there's no need for this additional condition:

```suggestion
    .keyboard-visible #reply-control.open {
```